projects/pahomqttc/patch.diff

```diff
@@ -0,0 +1,80 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
```

Comment:
Please get these landed in upstream repo, we cant accept these patches as they will break easily.

Reply:
https://github.com/eclipse/paho.mqtt.c/pull/1096